### PR TITLE
warn missing ruby version

### PR DESCRIPTION
**Pull request by s12chung**
just a nice to have, just like in `rbenv`

### CODE REVIEW

File: etc/rbenv.d/which/bundler-ruby-version.bash [hunk 1]
```diff
@@ -2,4 +2,9 @@ RBENV_BUNDLER_VERSION=$(rbenv-bundler-ruby-version)
 
 if [ -n "$RBENV_BUNDLER_VERSION" ]; then
   RBENV_COMMAND_PATH="${RBENV_ROOT}/versions/${RBENV_BUNDLER_VERSION}/bin/${RBENV_COMMAND}"
-fi
+
+  if ! [ -d "${RBENV_ROOT}/versions/${RBENV_BUNDLER_VERSION}" ]; then
+    echo "bundler-ruby-version: version \`$RBENV_BUNDLER_VERSION' is not installed" >&2
+    exit 1
+  fi
+fi
```
Comment (aripollak) [inline etc/rbenv.d/which/bundler-ruby-version.bash]: Please leave the newline at the end here.

Comment (s12chung) [inline etc/rbenv.d/which/bundler-ruby-version.bash]: sure. done.

**Comment by aripollak** [comment]
Thanks for the patch. Could you add a test for the new behavior into which.bats?

**Comment by s12chung** [comment]
no problem. i'm just not sure how to (it's my first bash script).

**Pull request merged**